nx-X11/lib/modules/lc/Utf8/Makefile.am

```diff
@@ -0,0 +1,28 @@
+NULL =
+
+noinst_LTLIBRARIES = libxlcUTF8Load.la
```

Comment:
>  If possible move that after AM_CFLAGS to make it consistent with upstream (and the om/generic Makefile.am.)

I'd rather not. The Makefile.am files have been changed considerably compared to upstream to provide better readability in nx-libs. This was done on purpose. Regarding the noinst_LTLIBRARIES field, I moved it up to have it visisble immediately when the file gets opened in en editor.